test/name.cpp

```diff
@@ -103,10 +104,10 @@ TEST_CASE("quicr::Name Arithmetic Tests")
 
 TEST_CASE("quicr::Name Byte Array Tests")
 {
-  std::vector<uint8_t> byte_arr(sizeof(quicr::Name::uint_type) * 2);
-  for (size_t i = 0; i < sizeof(quicr::Name::uint_type); ++i) {
+  std::vector<uint8_t> byte_arr(quicr::Name::size());
+  for (size_t i = 0; i < quicr::Name::size() / 2; ++i) {
```

Comment:
These functions named "size" confuse me, since I can't tell if this is the string size, the size of the "name" in octets, the size in bits, or whatever. Maybe there is a convention, but what causes this confusion for me is the fact that these are 128-bit values and we talk about them this way, but then it's called "name" and the output is an array of octets or hex in some cases.  Maybe my confusion is unwarranted.